examples/src/main/java/io/dapr/examples/secrets/README.md

```diff
@@ -46,98 +43,20 @@ Then get into the examples directory:
 cd examples
 ```
 
-### Setting Vault locally
-
-Before getting into the application code, follow these steps in order to set up a local instance of Vault. This is needed for the local instances. Steps are:
-
-1. To run the vault container locally run: 
-<!-- Docker is writing output to stderr ... -->
-
-<!-- STEP
-name: Start vault
-expected_stderr_lines:
-  - 'Creating network "secrets_default" with the default driver'
-sleep: 10 
--->
-
-```bash
-docker-compose -f ./src/main/java/io/dapr/examples/secrets/docker-compose-vault.yml up -d
-``` 
-
-<!-- END_STEP -->
-
-2. Run `docker ps` to see the container running locally: 
-
-```bash
-342d3522ca14        vault                      "docker-entrypoint.s…"         34 seconds ago        Up About
-a minute   0.0.0.0:8200->8200/tcp                               secrets_hashicorp_vault_1
-```
-Click [here](https://hub.docker.com/_/vault/) for more information about the container image for Hashicorp's Vault.
-
-### Create a secret in Vault
-Dapr's API for secret store only support read operations. For this sample to run, we will first create a secret via the Vault's cli commands:
-
-Export the `VAULT_ADDR` for vault CLI:
-```bash
-export VAULT_ADDR=http://127.0.0.1:8200/
-```
-
-Login to Hashicorp's Vault:
-
-<!-- STEP
-name: Vault login
-expected_stdout_lines:
-  - "Success! You are now authenticated. The token information displayed below"
-  - "token                myroot"
-env:
-  VAULT_ADDR: "http://127.0.0.1:8200/"
--->
-
-```bash
-vault login myroot
-```
-
-<!-- END_STEP -->
+### Creating a JSON secret file locally
```

Comment:
Please, change this to just explain the local json format expected. The file is already present and versioned in the repo.

Reply:
i explained the local json file format,but i just can't find the secret local file that already present and versioned in the repo.

Reply:
The file is in this PR. See the file above.